var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -0,0 +1,410 @@
+                '-DSEACASProj_ENABLE_SEACASFastq:BOOL=%s' % (
+                    'ON' if '+fastq' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASGen3D:BOOL=%s' % (
+                    'ON' if '+gen3d' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASGenshell:BOOL=%s' % (
+                    'ON' if '+genshell' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASGjoin:BOOL=%s' % (
+                    'ON' if '+gjoin' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASGrepos:BOOL=%s' % (
+                    'ON' if '+grepos' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASIoss:BOOL=%s' % (
+                    'ON' if '+ioss' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASMapvar:BOOL=%s' % (
+                    'ON' if '+mapvar' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASMapvar-kd:BOOL=%s' % (
+                    'ON' if '+mapvar-kd' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASMapvarlib:BOOL=%s' % (
+                    'ON' if '+mapvar' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASMapvarlib:BOOL=%s' % (
+                    'ON' if '+mapvar-kd' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASMat2exo:BOOL=%s' % (
+                    'ON' if '+mat2exo' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASNemesis:BOOL=%s' % (
+                    'ON' if '+nemesis' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASNemslice:BOOL=%s' % (
+                    'ON' if '+nemslice' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASNemspread:BOOL=%s' % (
+                    'ON' if '+nemspread' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASNumbers:BOOL=%s' % (
+                    'ON' if '+numbers' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASPLT:BOOL=%s' % (
+                    'ON' if '+plt' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSVDI:BOOL=%s' % (
+                    'ON' if '+svdi' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSlice:BOOL=%s' % (
+                    'ON' if '+slice' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSupes:BOOL=%s' % (
+                    'ON' if '+supes' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplib:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplibC:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASSuplibCpp:BOOL=%s' % (
+                    'ON' if '+suplib' in spec else 'OFF'),
+                '-DSEACASProj_ENABLE_SEACASTxtexo:BOOL=%s' % (
+                    'ON' if '+txtexo' in spec else 'OFF'),
+            ])
+
+        if '+kokkos' in spec:
+            if '+openmp' in spec:
```

Comment:
There is no openmp variant.

Reply:
I also need to remove, at some point, the internal kokkos snapshot and go to some method of git submodules instead.